### DISCUSSION TITLE
Fast refresh disabled with concurrent mode enabled and react experimental version

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#16100] [Help] [answered]
Error: https://github.com/vercel/next.js/blob/master/errors/react-version.md
Any way I can enable fast refresh?

**Comment by timneutkens** [comment, marked as answer]
It's not actually disabled, just gives the warning as we check for react lower than 16.10 (which did not support it).

**Reply by timneutkens** [reply]
Opened an issue for it: https://github.com/vercel/next.js/issues/16103
